src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  background: #fff;
+  border-radius: 5px;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 36px;
+    margin-left: 3px;
+    margin-right: 3px;
+  }
+
+  &__desc {
+    color: #060b35;
+    font-family: Roboto, sans-serif ;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+}
+
+.card__product {
+  color: #616070;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+
+    &:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+      margin-right: 4px;
+    }
+  }
+}
+
+.card__reviews-count {
+  color: #060b35;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 16px;
+
+  &__amount {
+    color: #060b35;
+    text-align: right;
+    font-family: Roboto, sans-serif;
```

Comment:
maybe consider moving `font-family` in some parent tag, you won't have to rewrite it 😉